packages/block-library/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+### New Features
+
+- Add level controls to Site Title block
```

Comment:
`level` is a bit confusing. IIUC, the control allows us to set the heading level (`<h1 />`-`<h6 />`). Let's maybe call it

```suggestion
- Add heading level controls to Site Title block.
```